torch/csrc/utils/byte_order.cpp

```diff
@@ -69,6 +69,13 @@ static inline void swapBytes64(void* ptr) {
   memcpy(ptr, &output, sizeof(uint64_t));
 }
 
+static inline uint8_t decodeUInt8(const uint8_t *data) {
+  // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+  uint8_t output;
+  memcpy(&output, data, sizeof(uint8_t));
+  return output;
```

Comment:
I just want to point out that this is completely unnecessary; as uint8 is byte sized you can just read it directly out of the pointer, no need to memcpy lol

Reply:
Removed.